### PR TITLE
Update DOC to state active or deleted users

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/remove-users.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ ms.assetid: 494bec9c-6654-41f0-920f-f7f937ea9723
 description: "In this article, you'll learn how to remove deleted users from SharePoint in different scenarios."
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/remove-users.md:25]:
```suggestion
description: "In this article, you'll learn how to remove users from SharePoint in different scenarios."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @salarson, kindly commit the additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**